daemons/concerns/leader_stats_helper.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
+require_relative './front_stats_constants.rb'
+
 module LeaderStatsHelper
-  LEADERS_LIMIT = 9
-  MAINNET = "mainnet"
-  NETWORKS = [MAINNET, "testnet"].freeze
+  include FrontStatsConstants
+
+  VALIDATOR_FIELDS_FOR_LEADER = %w[name account avatar_url].freeze
+  DC_FIELDS_FOR_LEADER = %w[location_latitude location_longitude country_iso_code]
```

Comment:
If you are putting all these constants into a separated file you should also include those two as well.